blueprint/core/blueprint_AppHarness.h

```diff
@@ -39,9 +40,20 @@ namespace blueprint
         void watch (const juce::File& f);
         void watch (const std::vector<juce::File>& fs);
 
+        /** Run the initial evaluation step and then watch for file changes. */
         void start();
+
+        /** Stop any running file watch. */
         void stop();
 
+        /** Run the initial evaluation step exactly once.
+         *
+         *  It may be helpful to use AppHarness::start in DEBUG builds when you want
+         *  hot reloading for development, then switch in RELEASE builds to using
+         *  AppHarness::once to ignore file watching when you don't need it.
```

Comment:
This might be over the top but maybe an actual `#if JUCE_DEBUG` switch example inline would be useful?